src/Tests/TableTestState.php

```diff
@@ -40,6 +46,7 @@ public function assertNotChanged(): void
     public function assertChangesEqualsFixture(string $fixture, bool $exportMode = false): void
     {
         $changes = $this->getChanges();
+        $exportMode = $this->globalExportMode || $exportMode;
```

Comment:
the `assertEqualsFixture` method already checking the `globalExportMode` property of the class https://github.com/RonasIT/laravel-helpers/blob/c5a92fd81c3093a3073506127aa5d1383a01c372/src/Traits/FixturesTrait.php#L104